filament/src/details/UboManager.cpp

```diff
@@ -39,7 +39,7 @@ using allocation_size_t = BufferAllocator::allocation_size_t;
 // FenceManager
 // ------------------------------------------------------------------------------------------------
 
-void UboManager::FenceManager::track(DriverApi& driver, std::unordered_set<AllocationId>&& allocationIds) {
+void UboManager::FenceManager::track(DriverApi& driver, std::vector<AllocationId>&& allocationIds) {
```

Comment:
you could use a `using AllocationIdContainer = std::vector<>` so that you can (almost) switch the container easily.